### PR TITLE
Markdown bugs

### PR DESCRIPTION
**Pull request by kof**
## Description

1. Don't handle global paste even when inside content-editable (before that when you paste inside texteditable, it would insert an instance additionally outside)
2. Insert links in paragraphs as RichTextLink
3. Make sure rich text link has the same props as link

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/shared/copy-paste/init-copy-paste.ts [hunk 1]
```diff
@@ -77,6 +79,9 @@ export const initCopyPaste = (options: Options) => {
   };
 
   const handlePaste = (event: ClipboardEvent) => {
+    if (textEditingInstanceIdStore.get()) {
+      return;
+    }
```
Comment (TrySound) [inline apps/builder/app/shared/copy-paste/init-copy-paste.ts:84]: Found more solid solution. Here we already check contenteditable as invalid element. Though the problem is that lexical catches it earlier.

We can fixed it by capturing event. Add `{ capture: true}` for paste event.

Comment (kof) [reply apps/builder/app/shared/copy-paste/init-copy-paste.ts:84]: I tried, but the problem is when I get the event, the target will be already removed from the dom and replaced by lexical, so I can't properly detect contenteditable.

Comment (kof) [reply apps/builder/app/shared/copy-paste/init-copy-paste.ts:84]: let me try some more tricks

Comment (TrySound) [reply apps/builder/app/shared/copy-paste/init-copy-paste.ts:84]: Yes, I get this too with current event. Though capture worked before lexical removed element from dom.

Comment (kof) [reply apps/builder/app/shared/copy-paste/init-copy-paste.ts:84]: you were right, we even have that check, I must have somehow removed the capture or something

File: apps/builder/app/shared/copy-paste/init-copy-paste.ts [hunk 2]
```diff
@@ -107,7 +102,9 @@ export const initCopyPaste = (options: Options) => {
     document.addEventListener("cut", handleCut);
   }
   if (onPaste) {
-    document.addEventListener("paste", handlePaste);
+    // Capture is required so we get the element before content-editable removes it
+    // This way we can detect when we are inside content-editable and ignore the event
+    document.addEventListener("paste", handlePaste, { capture: true });
```
Comment (TrySound) [inline apps/builder/app/shared/copy-paste/init-copy-paste.ts:107]: Probably need for removeEventListener too

Comment (kof) [reply apps/builder/app/shared/copy-paste/init-copy-paste.ts:107]: wow first time seeing that capture is required for removal as well

**Review by TrySound — APPROVED** [review]

**Pull request merged**